torch/csrc/jit/serialization/export.cpp

```diff
@@ -1218,6 +1219,35 @@ void GraphEncoder::EncodeTypeProto(
   }
 }
 
+static std::string tensorLEdata(const at::Tensor& t) {
```

Comment:
```suggestion
static std::string get_little_endian_data(const at::Tensor& t) {
```

Consider snake case to be consistent with this file; avoid abbreviations.